src/firejail/usage.c

```diff
@@ -42,7 +42,7 @@ static char *usage_str =
 	"    --blacklist=filename - blacklist directory or file.\n"
 	"    --build - build a whitelisted profile for the application.\n"
 	"    --build=filename - build a whitelisted profile for the application.\n"
-	"    -c - execute command and exit.\n"
+	"    -c - login shell compatibility option (has no effect).\n"
```

Comment:
if it's not meant to be run by the user and has no effect, I wouldn't even show it in the usage list.